src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
               }
             }
           }
+        },
+        "valuationComparisonData": {
+          "type": "array",
+          "title": "Valuation Comparison Data",
+          "description": "A collection of data used to compare the valuations (market values) of properties in the real estate or property market. It typically includes various types of information about properties, such as recently sold properties, properties currently on sale, property pricing estimates, property characteristics, and more.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "listingType": {
+                "title": "Listing Type",
+                "type": "string",
+                "enum": ["RecentlySold", "CurrentlyOnSale"]
+              },
+              "listingStatus": {
+                "title": "Listing Status ",
+                "type": "object",
+                "properties": {
+                  "type": {
+                    "type": "string",
+                    "enum": [
+                      "Sale",
```

Comment:
Should this be 'For Sale?' Does this status really matter as part of the comparable data anyway?

Reply:
we have removed status under ListingStatus